flink-connectors/pom.xml

```diff
@@ -54,6 +54,7 @@ under the License.
 		<module>flink-connector-kafka</module>
 		<module>flink-connector-gcp-pubsub</module>
 		<module>flink-connector-kinesis</module>
+		<module>flink-connector-aws-kinesis-data-streams</module>
```

Comment:
Nit: Next commit?

Reply:
Same as above, tests/build fail if `flink-connector-kinesis` can't access it's dependencies.